modules/renter/consts.go

```diff
@@ -34,4 +34,12 @@ var (
 		Standard: time.Minute * 15,
 		Testing:  10 * time.Second,
 	}).(time.Duration)
+
+	// maxChunkCacheSize determines the maximum number of chunks that will be
+	// cached in memory.
+	maxChunkCacheSize = build.Select(build.Var{
```

Comment:
This is an optimzation for the download-and-reupload algorithm: up to `maxChunkCacheSize` downloaded chunks (used for repair) will be cached in memory in order to reduce the amount of redundant downloading.